apps/Minecraft Java/install-64

```diff
@@ -7,51 +7,97 @@ function error {
   exit 1
 }
 
-#use the error function often!
-#If a certain command is necessary for installation to continue, then add this to the end of it:
-# || error 'reason'
-#example below:
+#install modprobe if not already installed
+if ! command -v modprobe >/dev/null;then
+  
+  if [ -f /usr/bin/apt ];then
+    sudo apt update
+    sudo apt install -y kmod || echo "Failed to install modprobe."
+  else
+    error "Failed to find any package manager to install modprobe."
+  fi
+fi
 
-#Download java and launcher
-mkdir ~/Minecraft && cd ~/Minecraft && wget https://github.com/chunky-milk/Minecraft/raw/main/setupMC.sh && chmod +x setupMC.sh && ./setupMC.sh || error 'Failed to install the launcher!'
+#install unzip if not already installed
+if ! command -v unzip >/dev/null;then
+  
+  if [ -f /usr/bin/apt ];then
+    sudo apt update
+    sudo apt install -y unzip || echo "Failed to install unzip."
+  else
+    error "Failed to find any package manager to install unzip."
+  fi
+fi
 
-#Move launcher to /usr/share/
-sudo mkdir /usr/share/minecraftjava && sudo mv launcher.jar /usr/share/minecraftjava/launcher.jar
+#install svn if not already installed
+if ! sudo command -v svn >/dev/null;then
+  
+  if [ -f /usr/bin/apt ];then
+    sudo apt update
+    sudo apt install -y subversion || echo "Failed to install subversion."
+  else
+    error "Failed to find any package manager to install subversion."
+  fi
+fi
 
-#Create desktop shortcut
-cd ~/.local/share/applications/
+#Run modprobe fuse
+sudo modprobe fuse
+
+cd ~
+
+#Download client and assorted java stuff
+wget https://github.com/gl91306/lunar/raw/master/lunarclient-2.7.3c-armv7l.AppImage
```

Comment:
This should be ` wget https://github.com/gl91306/lunar/raw/master/lunarclient-2.7.3c-aarch64.AppImage `